pkgs/python-by-name/psi4/package.nix

```diff
@@ -134,7 +135,7 @@ let
 in
 buildPythonPackage rec {
   pname = "psi4";
-  version = "1.9.1";
+  version = "1.10";
```

Comment:
The hash needs to be update.

Reply:
Urks, that one got lost in my git rebasing ...